### PR TITLE
CTW-1061/encounters-fqs

### PR DESCRIPTION
**Pull request by pmadria3**
This draft PR is to wire up the encounters resource. 

Query is slightly broken.

### CODE REVIEW

File: src/services/fqs/queries/fragments.ts
Comment (WesleyKapow) [inline src/services/fqs/queries/fragments.ts:1]: Don't do this now, as you'd hit tons of merge conflicts, but this file is huge! We should refactor to `queries/fragments/foo.ts`, e.g. `location.ts`, and `observation.ts`, etc.

File: src/fhir/fqs.ts
Comment (WesleyKapow) [inline src/fhir/fqs.ts:1]: This already exists in `fqs/client.ts`

File: src/fhir/encounters.ts [hunk 1]
```diff
@@ -2,33 +2,59 @@ import { getIncludedBasics } from "./bundle";
...
     [],
-    withTimerMetric(async (requestContext, patient) => {
-      try {
-        const { bundle, resources: encounters } = await searchCommonRecords(
-          "Encounter",
-          requestContext,
-          {
-            patientUPID: patient.UPID,
+    enableFQS
+      ? withTimerMetric(async (requestContext, patient) => {
```
Comment (WesleyKapow) [inline src/fhir/encounters.ts:15]: Split off into fqs and ods helper functions.

File: src/fhir/diagnostic-report.ts
Comment (WesleyKapow) [inline src/fhir/diagnostic-report.ts:1]: Anything not super specific to fhir should be moved to `services/diagnostic-reports.ts` to align with condition service.

File: src/components/content/zus-aggregated-profile/zus-aggregated-profile-tabs.tsx [hunk 2]
```diff
@@ -149,6 +149,6 @@ export const zusAggregatedProfileTabs: ZusAggregatedProfileTabs = {
         <BetaLabel />
       </div>
     ),
-    render: () => <PatientTimeline {...props} />,
+    render: () => <PatientTimeline {...props} enableFQS />,
```
Comment (WesleyKapow) [inline src/components/content/zus-aggregated-profile/zus-aggregated-profile-tabs.tsx:152]: This should be feature flagged.

File: src/components/content/observations/helpers/drawer.tsx [hunk 3]
```diff
@@ -35,7 +35,7 @@ export function ObservationsDrawer({
       showCloseFooter
     >
       <Drawer.Body>
-        <ObservationDetails diagnosticReport={diagnosticReport} />
+        <ObservationDetails diagnosticReport={diagnosticReport} enableFQS />
```
Comment (WesleyKapow) [inline src/components/content/observations/helpers/drawer.tsx:38]: Feature flag.

**Comment by pmadria3** [comment]
This PR is closed due to merge issues.

**Pull request closed without merging**